teammapper-backend/src/map/services/maps.service.ts

```diff
@@ -107,20 +108,60 @@ export class MapsService {
     return await this.mapsRepository.findOne({ where: { id: mapId }} );
   }
 
-  getDeletedAt(lastModified: Date, afterDays: number): Date {
+  async getDeletedAt(map: MmpMap, afterDays: number): Promise<Date> {
+    // get newest node of this map:
+    const newestNodeQuery = this.nodesRepository
+                                .createQueryBuilder()
+                                .select("max(MmpNode.lastModified) AS lastModified")
+                                .where( { nodeMapId: map.id } )
+    const newestNode = newestNodeQuery.getRawOne()
```

Comment:
why not ` .getOne()` ? that would also enable to use a proper object accessor instead of hash syntax

Reply:
unfortunately getOne doesn't work with max().

Reply:
if both are valid options for you, I would prefer the one with `getOne` as you can use normal object selectors after that - but if you think max is better, its also fine.